aten/src/ATen/native/cuda/IGammaKernel.cu

```diff
@@ -365,7 +365,7 @@ __host__ __device__ scalar_t _igamc_helper_continued_fraction(scalar_t a, scalar
 }
 
 template <typename scalar_t>
-inline __host__ __device__ scalar_t calc_igammac(scalar_t a, scalar_t x) {
+__host__ __device__ scalar_t calc_igammac(scalar_t a, scalar_t x) {
```

Comment:
Do you want to preemptively `__noinline__` this, too? It's about to be exposed directly to users.

Reply:
will do